common-ktx/src/test/java/co/nimblehq/common/extensions/DateExtTest.kt

```diff
@@ -0,0 +1,341 @@
+package co.nimblehq.common.extensions
+
+import org.junit.Assert.*
+import org.junit.Test
+import java.text.SimpleDateFormat
+import java.util.*
+
+class DateExtTest {
+
+    @Test
+    fun addition_isCorrect() {
```

Comment:
Can you please update this test case name to follow our naming convention for consistency? And the test case should be private? Please double-check all the test case from `DateRangeTest`, `IntExtTest` and `TimeIntervalTest` as well 🙏